lumispy/signals/common_luminescence.py

```diff
@@ -37,43 +38,13 @@
 
 class CommonLumi:
     """**General luminescence signal class (dimensionless)**"""
-
+    # Deprecated, to be removed for v1.0 release
```

Comment:
Thinking about it again, we do not need to deprecate the use here. Similarly like for `savetxt` and `to_array` a wrapper function can live in the `LumiSpectrum` class. It should take the same parameters as the new function, but `crop_px` in addition for backwards compatibility. A deprectation warning for that parameter is already implemented.